pom.xml

```diff
@@ -4,7 +4,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.mapfish.print</groupId>
   <artifactId>print-lib</artifactId>
-  <version>2.0-SNAPSHOT</version>
+  <version>geosolutions-2.0</version>
```

Comment:
I won't change this in master branch, we  need a branch / tag for the release

Reply:
I changed the base